app/src/docs/_examples/tables/ColumnFiltersTable.example.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useMemo, useState } from 'react';
+import { DataColumnProps, useLazyDataSource, useUuiContext, TableFiltersConfig, LazyDataSource, useTableState, DataTableState, getSeparatedValue } from '@epam/uui-core';
+import { Text, DataTable, Panel, defaultPredicates, FlexRow, Badge, EpamAdditionalColor } from '@epam/promo';
+import { Person } from '@epam/uui-docs';
+import css from './TablesExamples.module.scss';
+import { rangeDatePickerPresets } from '@epam/uui';
+import dayjs from 'dayjs';
+
+const personColumns: DataColumnProps<Person, number>[] = [
+    {
+        key: 'name',
+        caption: 'Name',
+        render: (p) => <Text>{p.name}</Text>,
+        width: 180,
+        isSortable: true,
+        isAlwaysVisible: true,
+    }, {
+        key: 'profileStatus',
+        caption: 'Profile status',
+        info: 'This person profile status information',
+        render: (p) =>
+            p.profileStatus && (
+                <FlexRow>
+                    <Badge fill="transparent" color={ p.profileStatus.toLowerCase() as EpamAdditionalColor } caption={ p.profileStatus } />
+                </FlexRow>
+            ),
+        width: 140,
+        isSortable: true,
+        isFilterActive: (f) => !!f.profileStatusId,
+    }, {
+        key: 'salary',
+        caption: 'Salary',
+        render: (p) => (
+            <Text>
+                {getSeparatedValue(+p.salary, {
+                    style: 'currency', currency: 'USD', maximumFractionDigits: 2, minimumFractionDigits: 2,
+                }, 'en-US')}
+            </Text>
+        ),
+        width: 150,
+        textAlign: 'right',
+        isSortable: true,
+    }, {
+        key: 'jobTitle',
+        caption: 'Title',
+        render: (r) => <Text>{r.jobTitle}</Text>,
+        width: 200,
+        isFilterActive: (f) => !!f.jobTitleId,
+    }, {
+        key: 'birthDate',
+        caption: 'Birth date',
+        render: (p) => p?.birthDate && <Text>{dayjs(p.birthDate).format('MMM D, YYYY')}</Text>,
+        width: 120,
+        isSortable: true,
+    }, {
+        key: 'hireDate',
+        caption: 'Hire date',
+        render: (p) => p?.hireDate && <Text>{dayjs(p.hireDate).format('MMM D, YYYY')}</Text>,
+        width: 120,
+        isSortable: true,
+    },
+];
+
+export default function ColumnsConfigurationDataTableExample() {
+    const { api } = useUuiContext();
+
+    const filtersConfig = useMemo<TableFiltersConfig<Person>[]>(
+        () => [
+            {
+                field: 'profileStatusId',
+                columnKey: 'profileStatus',
+                title: 'Profile status',
+                type: 'multiPicker',
+                isAlwaysVisible: true,
+                dataSource: new LazyDataSource({ api: api.demo.statuses }),
+                predicates: defaultPredicates.multiPicker,
```

Comment:
We don't show predicates in table filters now, let's remove them from all filters

Reply:
done